src/main/resources/application-test.yaml

```diff
@@ -1,45 +1,16 @@
-application:
-  group: zagrodnicy (Bycza-zagroda)
-  version: 0.5.0
-
 #Datasource
 spring:
-  banner:
-    location: classpath:banner.txt
   datasource:
     url: jdbc:mysql://localhost:3306/trackexpensesapptest?useUnicode=true&serverTimezone=UTC
     username: root
     password: root
-    driver-class-name: com.mysql.cj.jdbc.Driver
-    hikari:
-      connection-timeout: 500
-      maximum-pool-size: 5
 
   jackson:
     default-property-inclusion: non_empty
 
   liquibase:
     change-log: db/dbchangelog-test.yaml
     drop-first: true
-  #Jpa/Hibernate:
-  jpa:
-    open-in-view: false
-    generate-ddl: false
-    hibernate:
-      ddl-auto: none
-
```

Comment:
But here we have different hibernate strategy. Should it stay here, shouldn't?

Reply:
I think validation of the database schema with entities should take place in every profile. Correct me if I'm wrong.

Reply:
I don't think so, because integration tests have cleaning schema. So for every test case database will be created again.